terminus-ui/src/navigation/navigation.component.ts

```diff
@@ -217,11 +222,53 @@ export class TsNavigationComponent implements OnInit, AfterViewInit {
   @Input()
   public user!: TsUser;
 
+  /**
+   * Define the user name length
+   */
+  @Input()
+  public userNameLength: number = 20;
+
+  /**
+   * Define the user name tooltip position
+   */
+  @Input()
+  public userNameTooltipPosition: TsTooltipPositionTypes = 'below';
+
   /**
    * Define the welcome message
    */
   @Input()
-  public welcomeMessage: string = 'Welcome';
+  public welcomeMessage: string = 'Welcome,';
+
+  /**
+   * Define the welcome message length
+   */
+  @Input()
+  public welcomeMsgLength: number = 25;
+
+  /**
+   * Define the welcome message tooltip position
+   */
+  @Input()
+  public welcomeMsgTooltipPosition: TsTooltipPositionTypes = 'below';
+
+  /**
+   * Getter to return truncated user name
+   */
+  public get truncatedUserName(): string | undefined {
+    if (this.usersFullName && this.usersFullName.length > this.userNameLength) {
```

Comment:
We should probably tell our test coverage that we don't care about the 'else' path here:

```
  // istanbul ignore else
  if (...)
```

Same for the `if` in `truncatedWelcomeMsg` also